styles/about.css

```diff
@@ -0,0 +1,146 @@
+/* DEFAULT PAGE STYLING */
+body{
+  text-decoration:none;
+  color: rgb(255, 255, 255);
+  margin: 8px 15px 0px 15px;
+  font-family: 'Times New Roman', serif
+}
+
+/* BACKGROUND IMAGE */
+body, html {
+  height: 100%;
+}
+
+.bg {
+/* The image used */
+background-image: url("../images/image (1).jpg");
+
+/* Full height */
+height: 100%;
+
+/* Center and scale the image nicely */
+background-position: center center;
+background-repeat: no-repeat;
+background-size: cover;
+background-attachment: fixed;
+background-color: rgb(99, 115, 141);
+}
+
+/* NAVIGATION BAR */
+.navlinksmain {
+  font-size: 20px;
+  list-style-type: none;
+  float: left;
```

Comment:
Try to use `inline-block` rather than `float`. `float` has some side effects that can cause design issues.